tablo/models.py

```diff
@@ -740,14 +744,13 @@ def update_feature(self, feature):
                 else:
                     argument_values.append(feature['attributes'][key])
 
-        if feature['geometry']:
-            set_point_command = 'UPDATE {dataset_table_name} SET {point_column} = ST_Transform(ST_SetSRID(ST_MakePoint({long}, {lat}), 3857),{web_srid}) WHERE {primary_key}=%s'.format(
+        if feature.get('geometry'):
+            set_geom_command = 'UPDATE {dataset_table_name} SET {geom_column} = ST_Transform(ST_GeomFromEWKT(\'{geom}\'), {table_srid}) WHERE {primary_key}=%s'.format(
```

Comment:
It looks like this could move to just above line 770, and you can then drop this conditional here.